tests/tests_fabric/strategies/test_ddp.py

```diff
@@ -126,3 +130,27 @@ def test_ddp_grad_clipping(clip_type, accelerator, precision):
         clipping_test_cls = _MyFabricGradVal
     fabric = clipping_test_cls(accelerator=accelerator, devices=2, precision=precision, strategy="ddp")
     fabric.run()
+
+
+@RunIf(min_cuda_gpus=2)
+@pytest.mark.parametrize(
+    "precision,expected_dtype",
+    [
+        (Precision(), torch.float32),
+        (DoublePrecision(), torch.float64),
```

Comment:
https://github.com/Lightning-AI/lightning/pull/17287 will add the remaining options here